### PR TITLE
[docs] search algorithm notebook examples

### PR DESCRIPTION
**Pull request by ghost**
## Why are these changes needed?

We want to help new users become aware of, get started quicker with, and learn more effectively about Tune + search algorithms.

Checks
I've run scripts/format.sh to lint the changes in this PR.
I've included any doc changes needed for https://docs.ray.io/en/master/.

## Testing Strategy

Ran individual notebooks
Ran make html

### CODE REVIEW

File: doc/source/ray-more-libs/lightgbm-ray.md [hunk 1]
```diff
@@ -1,8 +1,572 @@
-<!--
-DO NOT EDIT THIS FILE
+(lightgbm-ray)=
```
Comment (maxpumperla) [inline doc/source/ray-more-libs/lightgbm-ray.md:2]: @thetwotravelers we need to revert these changes. same for xgboost below. other than that, outstanding contribution!

**Review by maxpumperla — CHANGES_REQUESTED** [review]
We need to revert changes to the three "ray-more-libs" files, then we're good to go. 👍

File: doc/source/ray-more-libs/lightgbm-ray.md (hunk 1 above)
Comment (thetwotravelers) [reply doc/source/ray-more-libs/lightgbm-ray.md]: thanks for catching this!

**Review by maxpumperla — APPROVED** [review]
